UDF/tg_ExprFunctions.hpp

```diff
@@ -0,0 +1,440 @@
+/******************************************************************************
+ * Copyright (c) 2015-2021, TigerGraph Inc.
+ * All rights reserved.
+ * Project: TigerGraph Query Language
+ * udf.hpp: a library of user defined functions used in queries.
+ *
+ * - This library should only define functions that will be used in
+ *   TigerGraph Query scripts. Other logics, such as structs and helper
+ *   functions that will not be directly called in the GQuery scripts,
+ *   must be put into "ExprUtil.hpp" under the same directory where
+ *   this file is located.
+ *
+ * - Supported type of return value and parameters
+ *     - int
+ *     - float
+ *     - double
+ *     - bool
+ *     - string (don't use std::string)
+ *     - accumulators
+ *
+ * - Function names are case sensitive, unique, and can't be conflict with
+ *   built-in math functions and reserve keywords.
+ *
+ * - Please don't remove necessary codes in this file
+ *
+ * - A backup of this file can be retrieved at
+ *     <tigergraph_root_path>/dev_<backup_time>/gdk/gsql/src/QueryUdf/ExprFunctions.hpp
+ *   after upgrading the system.
+ *
+ ******************************************************************************/
+
+#ifndef EXPRFUNCTIONS_HPP_
+#define EXPRFUNCTIONS_HPP_
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string>
+#include <gle/engine/cpplib/headers.hpp>
+#include <algorithm>
+#include <functional>
+#include <chrono>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#include <map>
+#include <random>
+#include <vector>
+
+/**     XXX Warning!! Put self-defined struct in ExprUtil.hpp **
+ *  No user defined struct, helper functions (that will not be directly called
+ *  in the GQuery scripts) etc. are allowed in this file. This file only
+ *  contains user-defined expression function's signature and body.
+ *  Please put user defined structs, helper functions etc. in ExprUtil.hpp
+ */
+#include "tg_ExprUtil.hpp"
+
+namespace tg_UDIMPL {
+  typedef std::string string; //XXX DON'T REMOVE
+
+  /****** BIULT-IN FUNCTIONS **************/
+  /****** XXX DON'T REMOVE ****************/
+  inline int str_to_int (string str) {
```

Comment:
I don't think we want to include these. These should be in ExprFunctions already and will likely result in conflict. I only see std::to_string used in this file, so removing them should not be problematic. Compile it to make sure once you remove them